### ISSUE TITLE
"Null value resolved for non-null field `expiresInQueueAt`" error

### ISSUE THREAD

**Issue by jellegerbrandy** [daostack/subgraph#347]
```query ProposalsSearchAllData
        {
          proposals (where: {

dao: "0xbe08030e687066fec4433f013953293167de68aa"

    }) {
            id

      expiresInQueueAt
          }
        }
  ```
results in 
```
{
  "errors": [
    {
      "locations": [
        {
          "column": 7,
          "line": 10
        }
      ],
      "message": "Null value resolved for non-null field `expiresInQueueAt`"
    }
  ]
}
```
on https://thegraph.com/explorer/subgraph/daostack/v28_0_rinkeby

**Comment by orenyodfat** [comment]
only on rinkeby and nectar dao

**Comment by jellegerbrandy** [comment]
same error for other fields (the query that alchemy is running is

```
 {
          proposals (where: {
      stage_in: ["Boosted","QuietEndingPeriod","Queued","PreBoosted"]
dao: "0xbe08030e687066fec4433f013953293167de68aa"

    }) {
            ...ProposalFields
          }
        }
        fragment ProposalFields on Proposal {
      id
      accountsWithUnclaimedRewards
      boostedAt
      confidenceThreshold
      contributionReward {
        id
        beneficiary
        ethReward
        externalToken
        externalTokenReward
        externalToken
        nativeTokenReward
        periods
        periodLength
        reputationReward
        alreadyRedeemedReputationPeriods
        alreadyRedeemedExternalTokenPeriods
        alreadyRedeemedNativeTokenPeriods
        alreadyRedeemedEthPeriods
      }
      createdAt
      dao {
        id
        schemes {
          id
          address
        }
      }
      description
      descriptionHash
      executedAt
      executionState
      expiresInQueueAt
      genericScheme {
        id
        contractToCall
        callData
        executed
        returnValue
      }
      genesisProtocolParams {
        activationTime
        boostedVotePeriodLimit
        daoBountyConst
        limitExponentValue
        minimumDaoBounty
        preBoostedVotePeriodLimit
        proposingRepReward
        queuedVotePeriodLimit
        queuedVoteRequiredPercentage
        quietEndingPeriod
        thresholdConst
        votersReputationLossRatio
      }
      gpRewards {
        id
      }
      scheme {
        id
        paramsHash
        name
        address
        canDelegateCall
        canManageGlobalConstraints
        canRegisterSchemes
        canUpgradeController
        name
      }
      gpQueue {
        id
        threshold
        votingMachine
      }
      organizationId
      preBoostedAt
      proposer
      quietEndingPeriodBeganAt
      schemeRegistrar {
        id
        schemeToRegister
        schemeToRegisterParamsHash
        schemeToRegisterPermission
        schemeToRemove
        decision
        schemeRegistered
        schemeRemoved
      }
      stage
      stakes {
        id
      }
      stakesFor
      stakesAgainst
      totalRepWhenCreated
      totalRepWhenExecuted
      title
      url
      votes {
        id
      }
      votesAgainst
      votesFor
      votingMachine
      winningOutcome
    }
      ```

**Comment by ben-kaufman** [comment]
I can't reproduce the issue. All queries work well. Should I close this issue?

**Comment by jellegerbrandy** [comment]
it was fixed with yesterday's update

**Comment by jellegerbrandy** [comment]
I have the exact same error now in my test environment, where a nec dao is employed using the params of the nectar dao repo.

Can you guys either release a new version (if the problem was a subgraph problem) or give me some instructions on how to resolve? 

@orenyodfat @ben-kaufman

**Comment by orenyodfat** [comment]
against which arc version the dao is deployed ? 
which migration version did you used ?  
please send a link to the test env

**Comment by jellegerbrandy** [comment]
I' m working on the ` NecDAO`  branch in the test_env, PR is here: https://github.com/daostack/test_env/pull/15

The latest images I created are these: https://github.com/daostack/client/blob/ugenericscheme/docker-compose.yml

The deployment of contracts and subgraph, and the indexing, occurs without errors - I only get an erro after creating a proposal.

**Comment by orenyodfat** [comment]
fix by #350
